src/Primitives.hs

```diff
@@ -415,16 +415,16 @@ primitiveMembers _ ctx target = do
     _ -> pure (evalError ctx ("Can't get the members of non-symbol: " ++ pretty target) (xobjInfo target))
   where
     env = contextEnv ctx
-    bottomedTarget =
-      case target of
+    bottomedTarget t =
+      case t of
         XObj (Sym targetPath _) _ _ ->
           case lookupBinder targetPath env of
             -- this is a trick: every type generates a module in the env;
             -- we’re special-casing here because we need the parent of the
             -- module
-            Just (Binder _ (XObj (Mod _) _ _)) -> target
+            Just (Binder _ (XObj (Mod _) _ _)) -> t
             -- if we’re recursing into a non-sym, we’ll stop one level down
```

Comment:
Seems like this comment no longer holds since we recurse again into the contents of the Binder (`x`) now

Reply:
It actually does, hence the comment about "one level down"—otherwise we should stop right here.

Reply:
Ah, you're right! -- I missed the catch all case below this line